### ISSUE TITLE
Sorting doesn't consider CD number

### ISSUE THREAD

**Issue by rrrnld** [apognu/otter#108]
**Describe the bug**

When a release contains multiple discs, the disc number is not taken into account when sorted. When only the track number is taken into account the sorted list mixes tracks appearing on subsequent discs.

**How to reproduce**

See above.

**Expected behavior**

Sorting order should be by disc-number and then track number (`1-01, 1-02, 2-01, 3-01, 3-02` and so forth).

**Environment details**

 - Device: Xiaomi Poco F1
 - Android version: 10
 - App version: 1.0.21
 - If public Funkwhale instance, its URL: -

**Comment by Thovi98** [comment]
Yes, it’s a known issue : try the latest develop apk which resolves it :) 
https://github.com/apognu/otter/releases/download/tip/app-debug.apk

**Comment by AMoonRabbit** [comment]
Yep, relates to #73
